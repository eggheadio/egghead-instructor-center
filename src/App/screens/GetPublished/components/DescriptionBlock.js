import React from 'react'

const DescriptionBlock = ({children}) => (
  <div className='i'>
    {children}
  </div>
)

DescriptionBlock.propTypes = {
  children: React.PropTypes.string.isRequired,
}

export default DescriptionBlock
