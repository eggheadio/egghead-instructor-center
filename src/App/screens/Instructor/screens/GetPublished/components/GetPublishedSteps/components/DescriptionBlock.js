import React from 'react'

const DescriptionBlock = ({children}) => (
  <div className='i'>
    {children}
  </div>
)

DescriptionBlock.propTypes = {
  children: React.PropTypes.node.isRequired,
}

export default DescriptionBlock
