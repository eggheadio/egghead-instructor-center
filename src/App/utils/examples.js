export default {
  text: {
    short: 'Lorem ipsum',
    medium: 'Lorem ipsum dolor sit amet, urna erat feugiat egestas, metus molestie, adipiscing sed nisl et, dictumst erat sem erat.',
  },
  url: 'https://egghead.io',
  markdown: 'Lorem ipsum *dolor*',
}
