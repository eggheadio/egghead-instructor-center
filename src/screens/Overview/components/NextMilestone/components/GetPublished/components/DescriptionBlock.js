import React, {PropTypes} from 'react'

const DescriptionBlock = ({children}) => (
  <div className='i'>
    {children}
  </div>
)

DescriptionBlock.propTypes = {
  children: PropTypes.node.isRequired,
}

export default DescriptionBlock
