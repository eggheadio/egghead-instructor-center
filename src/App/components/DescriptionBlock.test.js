import React from 'react'
import renderer from 'react-test-renderer'

import examples from '../../App/utils/examples'

import DescriptionBlock from './DescriptionBlock'

test('Default', () => {
  const tree = renderer.create(
    <DescriptionBlock>
      {examples.text.short}
    </DescriptionBlock>
  )
  expect(tree).toMatchSnapshot()
})
