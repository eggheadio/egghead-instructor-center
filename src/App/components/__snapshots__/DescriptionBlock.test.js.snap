exports[`test Default 1`] = `
<div
  className="i">
  Lorem ipsum
</div>
`;
